dataland-frontend/src/components/resources/dataTable/cells/ModalLinkDisplayComponent.vue

```diff
@@ -1,15 +1,8 @@
 <template>
-  <a
-    v-if="
-      content?.displayValue?.modalOptions?.data?.listOfRowContents?.length ||
-      content?.displayValue?.modalOptions?.data?.values?.length
```

Comment:
Why is this removed?

Reply:
These are checks that make framework-specific assumptions about the data-contents of the modal. It took me ~2h to figure out why my new modal was not being displayed. These checks should not be handled here, but rather in the ValueGetterFactories of the field (I added them there).

Reply:
ok, closed